classes/module/Module.php

```diff
@@ -373,6 +373,19 @@ public function __construct($name = null, ?Context $context = null)
     public function install()
     {
         Hook::exec('actionModuleInstallBefore', ['object' => $this]);
+
+        PrestaShopLogger::addLog(
+            Context::getContext()->getTranslator()->trans(
+                'Starting module install: %s v%s',
+                [$this->name, $this->version],
+                'Admin.Modules.Notification'
+            ),
+            1,
```

Comment:
```suggestion
            PrestaShopLogger::LOG_SEVERITY_LEVEL_INFORMATIVE
```

Reply:
Can you please use the const everywhere so we can understand right away the level of severity?

Reply:
Done thnks @jolelievre